CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
 * [#1765](https://github.com/bbatsov/rubocop/pull/1765): Update 1.9 hash to stop triggering when the symbol is not valid in the 1.9 hash syntax. ([@crimsonknave][])
 * [#1806](https://github.com/bbatsov/rubocop/issues/1806): Require a newer version of `parser` and use its corrected solution for comment association in `Style/Documentation`. ([@jonas054][])
 * [#1792](https://github.com/bbatsov/rubocop/issues/1792): Fix bugs in `Sample` that did not account for array selectors with a range and passing random to shuffle. ([@rrosenblum][])
+* [#1767](https://github.com/bbatsov/rubocop/pull/1767): Add more acceptable methods to `Rails/TimeZone` (`utc`, `localtime`, `to_i`, `iso8601` etc). ([@palkan][])
```

Comment:
I'll fix this.

On 20 April 2015 at 20:57, Vladimir Dementyev <notifications@github.com>
wrote:

> In CHANGELOG.md
> <https://github.com/bbatsov/rubocop/pull/1787#discussion_r28712332>:
>
> > @@ -15,6 +15,7 @@
> >  * [#1765](https://github.com/bbatsov/rubocop/pull/1765): Update 1.9 hash to stop triggering when the symbol is not valid in the 1.9 hash syntax. ([@crimsonknave][])
> >  * [#1806](https://github.com/bbatsov/rubocop/issues/1806): Require a newer version of `parser` and use its corrected solution for comment association in `Style/Documentation`. ([@jonas054][])
> >  * [#1792](https://github.com/bbatsov/rubocop/issues/1792): Fix bugs in `Sample` that did not account for array selectors with a range and passing random to shuffle. ([@rrosenblum][])
> > +* [#1767](https://github.com/bbatsov/rubocop/pull/1767): Add more acceptable methods to `Rails/TimeZone` (`utc`, `localtime`, `to_i`, `iso8601` etc). ([@palkan][])
>
> Oops, sorry! Should I add another PR to fix this?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bbatsov/rubocop/pull/1787/files#r28712332>.
>



-- 
Best Regards,
Bozhidar Batsov

http://www.batsov.com